commands/kubernetes.go

```diff
@@ -477,6 +480,37 @@ This command deletes the specified node in the specified node pool, and then cre
 	return cmd
 }
 
+func kubernetesRegistryIntegration() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "registry",
+			Aliases: []string{"reg"},
+			Short:   "Display commands for integrating clusters with docr",
+			Long:    "The commands under `registry` are for managing DOCR integration with Kubernetes clusters. You can use these commands to add or remove registry from one or more clusters.",
+		},
+	}
+
+	k8sCmdService := kubernetesCommandService()
+
+	cmdRegistryAdd := CmdBuilder(cmd, k8sCmdService.RunKubernetesRegistryAdd,
+		"add", "Add registry for clusters", `
+This command add registries to the specified cluster(s).`,
+		Writer, aliasOpt("a"))
+
+	AddStringSliceFlag(cmdRegistryAdd, doctl.ArgClusterUUID, "", nil,
+		"uuid of cluster to add registry for; repeat to specify additional clusters.")
```

Comment:
A couple thoughts here.

It might be a bit nicer UX to use arguments rather than flags here. Eg:

```
doctl kubernetes cluster registry add <uuid> <uuid>
```

rather than:
```
doctl kubernetes cluster registry add --cluster-uuid <uuid> --cluster-uuid <uuid>
```



Additionally, other Kubernetes commands take either a name or UUID. Might be nice be consistent with them here. There is a `clusterIDize` helper:

https://github.com/digitalocean/doctl/blob/51e58c7cf399ba128622352642892a1000725813/commands/kubernetes.go#L1808